Contributor/En/Guide.xyl

```diff
@@ -392,6 +392,40 @@ $ git commit</code></pre>
   <p>Do not edit the <code>CHANGELOG.md</code> file. It is automatically
   generated.</p>
 
+  <h3 id="Commit_message_structure" for="main-toc">Commit message structure</h3>
+
+  <p>Has we already seen in the previous section, a commit title must be 50
```

Comment:
a commit title must be **less than** 50 ?

Reply:
Yes it's clearer, I've updated the sentence.